docs/DeveloperGuide.md

```diff
@@ -450,6 +450,18 @@ We propose to make the index error messages more specific and highlight to the u
 that index is wrong. For example, in the `enrol m/1 e/-1` input, we will show an error message to the user along 
 the lines of "The provided Event Index is not a non-zero unsigned integer."
 
+### Make UI stay on current view upon undo/redo
+**Current Implementation:**
+* **Current Issue:** Currently, the `undo` and `redo` commands will always result in all members and events being listed.
+* **Example:** For example, if the user enrols a member to an event and views the member, before deciding to unenrol the member
+from the event by executing the `undo` command, all the members and events would then be listed. This inconveniences the user
+as they will then have to execute the `viewMember` command again to see their changes. 
+
+**Proposed solution:**
```

Comment:
I believe there is currently an PR to change `Proposed solution` to `Proposed Solution`, should we standardise it here now?

Reply:
Yes agreed.

Reply:
Done!